php/geocoordinates.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+//This file is autogenerated. See modules.json and autogenerator.py for details
+
+/*
+    geocoordinates.php
+
+    MediaWiki API Demos
+    Demo of `Geosearch` module: Obtain coordinates for wiki pages nearby
+
+    MIT License
+*/
+
+$endPoint = "https://en.wikipedia.org/w/api.php";
+$params = [
+    "action" => "query",
+    "prop" => "coordinates",
+    "titles" => "Wikimedia Foundation",
+    "format" => "json"
+];
+
+$url = $endPoint . "?" . http_build_query( $params );
+
+$ch = curl_init( $url );
+curl_setopt( $ch, CURLOPT_RETURNTRANSFER, true );
+$output = curl_exec( $ch );
+curl_close( $ch );
+
+$result = json_decode( $output, true );
+var_dump( $result );
```

Comment:
Here, we also obtain geocoordinates just like in: https://github.com/wikimedia/mediawiki-api-demos/blob/5d6745ecd1cb61dcbbdb5b96547329e589dff390/python/geocoordinates.py#L31.